### PR TITLE
Hello_Microverse_project

### PR DESCRIPTION
**Pull request by Surafels**
I created this project using the following:

1. Style.css file
2. Index.html file
3. readme.md

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,232 +27,85 @@ After you're finished please remove all the comments and instructions!
...
 - [📝 License](#license)
-
-<!-- PROJECT DESCRIPTION -->
-
-# 📖 [your_project_name] <a name="about-project"></a>
-
-> Describe your project in 1 or 2 sentences.
-
-**[your_project__name]** is a...
+# 📖 [project_name] <a name="Hello_Microverse_project"></a>
```
Comment (bienvenuushindi) [inline README.md:30]: :red_circle: Please make sure you include the proper project name here. Please replace the placeholder with the actual project name.

**Review by bienvenuushindi — CHANGES_REQUESTED** [review]
### Hi @Surafels      ,

Good job implementing most of the project requirements!
There are some issues that you still need to work on to go to the next project but you are almost there!



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Great start ✔✔✔✔

## (Changes Required) Aspects to improve: ♻️
- [ ] Please make sure you [set up linters ](https://github.com/microverseinc/linters-config)for this project. Kindly consider these comments when setting up linters for your repo
  - Linter should be detected by the Guthu Action
  - The linter result should be green. AKA: Linter should report NO errors.
Note:  The linters are not detected 

- [ ] Please check the inline comments below.

## [Optional] suggestions:

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md
Comment (AbelG101) [inline README.md:1]: - [ ] [OPTIONAL] I'd kindly suggest you fix the markdown formating as shown below in the screenshot as it's making your project look unprofessional and bit reckless. 😟
  image

**Review by AbelG101 — APPROVED** [review]
# Hi @Surafels 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct use of GitHub Flow ✔
-  Well-structured files ✔
- All linter checks are passing ✔
- Descriptive PR title with a good summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**